src/routes/api/tripRoute.js

```diff
@@ -421,4 +429,32 @@ tripRouter
     operateAcceptOrReject, acceptOrRejectRequestsController
   );
 
+/**
+ * @swagger
+ *
+ * /trips/mostTraveled:
+ *    get:
+ *      summary: get info about most traveled destination
+ *      tags: [Trip]
+ *      parameters:
+ *        - name: token
+ *          in: header
+ *          description: enter token
+ *          required: true
+ *          schema:
+ *            type: string
+ *      responses:
+ *        "200":
+ *          description:  Most traveled destination info retrieved successfully
+ *        "401":
+ *          description: No token provided
+ *        "404":
+ *          description: No Trip found
+ */
+
+tripRouter
+  .get(
+    '/mostTraveled', verifyToken, isUserVerified, mostTraveledDestination
+  );
+
```

Comment:
@william000000  Though this format is also valid, I prefer you to use the below format 
```
tripRouter
  .get(
    '/mostTraveled', 
    verifyToken, 
    isUserVerified,
    mostTraveledDestination
  );
```